projects/sbb-esta/angular-business/src/lib/checkbox/checkbox/checkbox.component.html

```diff
@@ -1,18 +1,22 @@
 <label [attr.for]="inputId">
   <input
+    #input
     type="checkbox"
     class="cdk-visually-hidden"
     [id]="inputId"
-    [value]="value"
+    [attr.name]="name"
+    [attr.value]="value"
     [required]="required"
     [checked]="checked"
     [indeterminate]="indeterminate"
-    [attr.aria-checked]="ariaChecked"
+    [tabIndex]="tabIndex"
+    [attr.aria-checked]="checked"
```

Comment:
Should probably be

```html
[attr.aria-checked]="ariaChecked"
```

or something else, if the checkbox has state indeterminate it should be mixed.

Reply:
Thanks. I missed that.